data-jdbc/src/test/groovy/io/micronaut/data/jdbc/oraclexe/jsonview/entities/StudentScheduleView.java

```diff
@@ -1,17 +1,24 @@
-package io.micronaut.data.jdbc.oraclexe.jsonview;
+package io.micronaut.data.jdbc.oraclexe.jsonview.entities;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.micronaut.data.annotation.Id;
+import io.micronaut.data.annotation.JsonData;
+import io.micronaut.data.annotation.JsonViewColumn;
 import io.micronaut.data.annotation.Relation;
+import io.micronaut.data.jdbc.annotation.JoinColumn;
+import io.micronaut.data.jdbc.oraclexe.jsonview.repositories.StudentScheduleClassView;
 import io.micronaut.serde.annotation.Serdeable;
 
 @Serdeable
+@JsonData(table = "TBL_STUDENT_CLASSES", permissions = "INSERT UPDATE DELETE")
 public class StudentScheduleView {
     @Id
     private Long id;
 
     @JsonProperty("class")
     @Relation(Relation.Kind.ONE_TO_ONE)
+    @JoinColumn(referencedColumnName = "CLASS_ID", name = "ID")
+    @JsonViewColumn(name = "class")
```

Comment:
Why not to use `@MappedProperty` to define the column name?

Reply:
We could use that, but needed to introduce `@JsonColumnView` to mark if column has attributes/permissions `INSERT UPDATE DELETE CHECK` so thought to use it for this as well.